presto-docs/src/main/sphinx/release/release-0.184.rst

```diff
@@ -0,0 +1,41 @@
+=============
+Release 0.184
+=============
+
+General Changes
+---------------
+
+* Fix query execution failure for ``split_to_map(...)[...]``.
+* Fix issue that caused queries containing ``CROSS JOIN`` to not release CPU resources
+  even after they are killed.
+* Fix planning failure due to incorrect static evaluation of aggregation queries
+  with ``count(*)`` and non empty ``GROUP BY`` clause.
+* Fix communication failures caused by lock contention in new scheduler.
+* Add :func:`hamming_distance` function.
+* Improve performance of :func:`element_at` to be constant time rather than O(n).
+* Improve performance of queries with gathering exchanges.
+
+Web UI Changes
+--------------
+
+* Add worker status page, with information about resource utilization (CPU, heap,
+  memory pools), and currently running threads. This page is accessible by clicking a
+  hostname on a query task list.
+
+Hive Changes
+------------
+
+* Reduce system memory usage when reading table columns containing string values
+  from ORC or DWRF files. This can prevent high GC overhead or out-of-memory crashes.
+* Fix partition filtering for keys of CHAR, DECIMAL, or DATE type.
+
+TPCDS Changes
+-------------
+
+* Fix statistics retrieval for ``TPCDS``  tables when using ``SHOW STATS FOR ...``.
```

Comment:
What was fixed?

Reply:
Don't know. I asked @kokosing

Reply:
It is about this change: https://github.com/prestodb/presto/commit/9ba6eaf1a80f0fa550dec044177374a0b38daf13
Previously statistics data returned from TPCDS connector were passed to planner without ny conversion. Now it is converted to Presto internal data type format. As it is not yet used anywhere, user could only observe that when running `SHOW STATS....` queries. 

I am sorry but I did not know how to explain that on a higher level.